docs/internals/optimizer.rst

```diff
@@ -323,18 +323,8 @@ on the individual steps and their sequence below.
 Selecting Optimizations
 -----------------------
 
-By default the optimizer applies its predefined sequence of optimization steps to
-the generated assembly. You can override this sequence and supply your own using
-the ``--yul-optimizations`` option:
-
-.. code-block:: bash
-
-    solc --optimize --ir-optimized --yul-optimizations 'dhfoD[xarrscLMcCTU]uljmul'
-
-The sequence inside ``[...]`` will be applied multiple times in a loop until the Yul code
-remains unchanged or until the maximum number of rounds (currently 12) has been reached.
-
-Available abbreviations are listed in the :ref:`Yul optimizer docs <optimization-step-sequence>`.
+Detailed information regrading the optimization sequence as well a list of abbreviations is
+available in the :ref:`Yul optimizer docs <optimization-step-sequence>`.
```

Comment:
This is the optimizer docs :) `yul.rst` documents the assembly (i.e. Yul):
```suggestion
available in the :ref:`assembly documentation <optimization-step-sequence>`.
```

Reply:
Not sure I get what you mean - was this wrong to begin with, because I've made no changes here? Also, this links to `yul.rst` `Optimization Step Sequence` anchor.

Reply:
True. Looks like it was wrong before. Or maybe not exactly wrong - I see now that Yul docs just have a small section about the optimizer so it's technically correct - but a bit misleading since the main optimizer docs are actually here, not there.

Looking at it now, I think it's a bit weird to have that section with content that's not present in the main optimizer docs. I think we should move most of its content to the main optimizer docs and leave it just as a quick stub with a link.